### DISCUSSION TITLE
Couple ideas after testing the app

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/TauonMusicRemote#2] [Ideas]
1. I think Welcome tab should be in the Settings
2. Same UI/logic as a desktop version, e.g. there is no playlist with all tracks of the selected playlist, just tracks from selected album
3.  Playlists can be in the left side menu (Navigation Drawer)

>Play users local music files?

interesting :thinking:

**Comment by Taiko2k** [comment]
Those are good ideas. Unfortunately I'm no longer actively working on this, so development is kind of suspended unless someone wants to submit a PR. 

Though for the about of work needed someone would more likely want to work on their own app. You could take a look at [sultannamja's one](https://github.com/sultannamja/tauonremote_android/releases). It has more or less those ideas you mentioned, though is currently in an earlier development stage and remote control only.

**Reply by Solarunit** [reply]
>Unfortunately I'm no longer actively working on this

oh
